Sources/WalletConnectKMS/Keychain/KeychainStorage.swift

```diff
@@ -100,11 +101,46 @@ public final class KeychainStorage: KeychainStorageProtocol {
     private func buildBaseServiceQuery(for key: String) -> [CFString: Any] {
         return [
             kSecClass: kSecClassGenericPassword,
-            kSecAttrAccessible: kSecAttrAccessibleWhenUnlockedThisDeviceOnly,
+            kSecAttrAccessible: kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly,
             kSecAttrIsInvisible: true,
             kSecUseDataProtectionKeychain: true,
             kSecAttrService: service,
             kSecAttrAccount: key
         ]
     }
+
+    private func tryMigrateAttrAccessible(key: String) throws -> Data? {
+        var query = buildBaseServiceQuery(for: key)
+        query[kSecReturnData] = true
+        query[kSecAttrAccessible] = kSecAttrAccessibleWhenUnlockedThisDeviceOnly
+
+        var item: CFTypeRef?
+        let status = secItem.copyMatching(query as CFDictionary, &item)
+
+        switch status {
+        case errSecSuccess: // Migration needed
+            guard let data = item as? Data else { return nil }
+            
+            // Fetching old value
+            var deleteQuery = buildBaseServiceQuery(for: key)
+            deleteQuery[kSecAttrAccessible] = kSecAttrAccessibleWhenUnlockedThisDeviceOnly
```

Comment:
if read failed for `kSecAttrAccessibleWhenUnlockedThisDeviceOnly` why delete won't